examples/source/multimedia-animations/Animated_hamburger_menu.html

```diff
@@ -0,0 +1,127 @@
+author: CrystalOnScript
+ 
+--->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Animated hamburger menu</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script><!-- ## Introduction --> 
+  <!--
+    Small interactive UI animations communicate state and purpose to users. A common case is a hamburger menu button that animates into an "X" when open. This animation communicates that the user can press the same button again to close the menu. 
+  -->
+  <link rel="canonical" href="<% canonical %>">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <!-- Custom styles and CSS animations for the hamburger menu: -->
+    <style amp-custom>
+      .hamburger_wrapper {
+        padding: 5px;
+        z-index: 500;
+      }
+
+      #hamburger {
+        width: 60px;
+        height: 45px;
+        position: relative;
+        transform: rotate(0deg);
+        transition: .5s ease-in-out;
+        cursor: pointer;
+        outline: none;
+
+      }
+      #hamburger span {
+        display: block;
+        position: absolute;
+        height: 9px;
+        width: 100%;
+        background: #0479C4;
+        border-radius: 9px;
+        opacity: 1;
+        left: 0;
+        transform: rotate(0deg);
+        transition: .5s ease-in-out;
+      }
+
+      #hamburger span:nth-child(1) {
+        top: 0px;
+        transform-origin: left center;
+      }
+
+      #hamburger span:nth-child(2) {
+        top: 21px;
+        transform-origin: left center;
+      }
+
+      #hamburger span:nth-child(3) {
+        top: 42px;
+        transform-origin: left center;
+      }
+
+      #hamburger.close span:nth-child(1) {
+        transform: rotate(45deg);
+      }
+
+      #hamburger.close span:nth-child(2) {
+        width: 0%;
+        opacity: 0;
+        transition: .1s;
+      }
+
+      #hamburger.close span:nth-child(3) {
+        transform: rotate(-45deg);
+      }
+      
+      .not-active {
+        position: relative;
+        transform: translateX(-100%);
+        opacity: 0;
+        z-index: 499;
+        transition: transform .5s ease, opacity ease .2s;
+      }
+      .now-active {
```

Comment:
When we define the 'not active' styles by using an id selector we also need to use the id here.
So the selector would be `#nav-menu.now-active`
(Or maybe simply use `active` instead of `now-active`)

Reply:
`active` appears to be a class already used by amp.dev and it applies the colors to the side menu, lol.

Reply:
Hehe, then lets keep `now-active`